autoload/neomake/makers/ft/text.vim

```diff
@@ -1,9 +1,11 @@
-function! neomake#makers#ft#text#EnabledMakers()
-    return ['proselint']
+function! neomake#makers#ft#text#EnabledMakers() abort
+    return ['proselint', 'writegood']
 endfunction
 
-function! neomake#makers#ft#text#proselint()
-    return {
-                \ 'errorformat': '%f:%l:%c: %m'
-                \ }
+function! neomake#makers#ft#text#writegood() abort
+    return neomake#makers#ft#markdown#writegood()
+endfunction
+
+function! neomake#makers#ft#text#proselint() abort
+    return neomake#makers#ft#markdown#proselint()
```

Comment:
`neomake#makers#ft#text#proselint` sneaked in here?!
Please remove it - it will be enhanced in https://github.com/neomake/neomake/pull/775/files#diff-52c614cc1832d5fcf376482ab60267d7.